composer/yamls/hwbenchmark/bert-stream.yaml

```diff
@@ -0,0 +1,75 @@
+train_dataset:
```

Comment:
We can probably leave the YAMLs out of the PR.. does it "just work" like this? If so, maybe we don't even need a PR then for integration? Maybe a notebook tutorial suffices?

Reply:
I can add a notebook later but I'd like to have this folder in the yamls for all the hw benchmarking (not just amd) i'm doing.